src/consensus/pbft/libbyz/Replica.cpp

```diff
@@ -1075,6 +1101,18 @@ void Replica::emit_signature_on_next_pp(int64_t version)
   signed_version = version;
 }
 
+void Replica::activate_pbft_local_hooks()
```

Comment:
Question: why is this required? It seems that it is only used in the unit test.

Reply:
Will be used in next PR where playback will happen via State messages. When we are not playing back from the ledger we don’t want the hooks to be triggered but we want execution to go on as usual